Arrowgene.Ddon.GameServer/Handler/JobJobValueShopBuyItemHandler.cs

```diff
@@ -48,8 +48,6 @@ public override S2CJobJobValueShopBuyItemRes Handle(GameClient client, C2SJobJob
             List<CDataItemUpdateResult> itemUpdateResults = _itemManager.AddItem(Server, client.Character, sendToItemBag, boughtListing.ItemId, boughtAmount);
             boughtAmount = (uint)itemUpdateResults.Select(result => result.UpdateItemNum).Sum();
 
-            var totalPrice = boughtAmount * packet.Price;
```

Comment:
Seems the total price for this was calculated by the client already, so we were charging the player more PP than was expected.

For example, I bought 15 items for 500 PP a piece in a single transaction which comes out to 7500 pp total. Yet in the debugger we were try to charge the player 112500 PP. This is because we did 15 * 7500 because the math expected that we were only given the price of 1 item, not the whole transaction.

![image](https://github.com/user-attachments/assets/836a1e72-20e3-4062-bcaa-0ed348dc987e)